### ISSUE TITLE
Rethink the PR template

### ISSUE THREAD

**Issue by oscerd** [apache/camel-quarkus#1015]
The text is too long. Nobody will ever read it and fill the square.

**Comment by davsclaus** [comment]
Yeah keep it low, one line with some gentle minder where to find more details about contributing
